ross/point_mass.py

```diff
@@ -0,0 +1,48 @@
+"""Point mass module.
+
+This module defines the PointMass class which will be used to link elements.
+"""
+import numpy as np
+from .element import Element
+
+__all__ = ["PointMass"]
+
+
+class PointMass(Element):
+    """A point mass element.
+
+    This class will create a point mass element.
+    This element can be used to link other elements in the analysis.
+
+    Parameters
+    ----------
+    n: int
+        Node which the bearing will be located in.
+    m: float
+        Mass for the element.
+    """
+
+    def __init__(self, n=None, m=None):
```

Comment:
I would include the option of informing two different masses, like Mx and My. In some cases the support inertia can be different in X and Y directions.
This would be quite simple to implement. Just include optional arguments `Mx` and `My` that will be used to built the matrix M. If only the argument `m` is provided, then we assign `Mx = My = m`.

Reply:
Thanks for the suggestion!
This was included on commit 6392579